src/axom/primal/geometry/Polyhedron.hpp

```diff
@@ -388,14 +392,18 @@ class Polyhedron
     return PointType(sum);
   }
 
-private:
+public:
   /*!
-   * \brief Finds the faces of the Polyhedron, assuming the vertex neighbors
-   *        are in counter-clockwise ordering.
+   * \brief Helper function to find the faces of the Polyhedron, assuming the
+   *        vertex neighbors are in counter-clockwise ordering.
+   *
+   * \param [out] faces is the vertex indices for faces
+   * \param [out] face_offset is the offset for each face
+   * \param [out] face_size is the number of vertices for each face
+   * \param [out] face_count is the number of faces
    *
```

Comment:
Please add a note that it is the user's responsibility to ensure that the provided buffers are sufficiently large and that the user must know these sizes ahead of time.  

`primal::Polyhedron` does not yet have a way to let the user know how big these arrays need to be.

Reply:
Technically, since we assume this is a convex polyhedron, Euler's formula tells us that $|v|-|e|+|f|=2$.
Since we know the number of vertices, $|v|$, and can calculate the number of edges, $|e|$, we can add a function to return the number of faces, $|f|$.

We should probably do that in a follow up PR.